lib/pure/collections/lists.nim

```diff
@@ -880,3 +954,31 @@ proc remove*[T](L: var DoublyLinkedRing[T], n: DoublyLinkedNode[T]) =
       L.head = nil
     else:
       L.head = L.head.prev
+
+proc append*[T](a: var (SinglyLinkedList[T] | SinglyLinkedRing[T]),
+                b: SinglyLinkedList[T] | SinglyLinkedNode[T] | T) =
+  ## Alias for `a.add(b)`.
+  ##
+  ## See also:
+  ## * `add proc <#add,SinglyLinkedList[T],SinglyLinkedNode[T]>`_
+  ## * `add proc <#add,SinglyLinkedList[T],T>`_
+  ## * `add proc <#add,T,T>`_
+  a.add b
+
+proc append*[T](a: var (DoublyLinkedList[T] | DoublyLinkedRing[T]),
+                b: DoublyLinkedList[T] | DoublyLinkedNode[T] | T) =
+  ## Alias for `a.add(b)`.
+  ##
+  ## See also:
+  ## * `add proc <#add,DoublyLinkedList[T],DoublyLinkedNode[T]>`_
+  ## * `add proc <#add,DoublyLinkedList[T],T>`_
+  ## * `add proc <#add,T,T>`_
+  a.add b
+
+proc appendMoved*[T: SomeLinkedList](a, b: var T) {.since: (1, 5, 1).} =
+  ## Alias for `a.addMoved(b)`.
+  ##
+  ## See also:
+  ## * `addMoved proc <#addMoved,SinglyLinkedList[T],SinglyLinkedList[T]>`_
+  ## * `addMoved proc <#addMoved,DoublyLinkedList[T],DoublyLinkedList[T]>`_
+  a.addMoved b
```

Comment:
```suggestion
```

refs https://github.com/nim-lang/Nim/pull/16536#issuecomment-753394613

Reply:
I think it would be a mistake not to add `appendMoved`, especially now that we are not deprecating the `append` functions at all. If there is `prependMoved`, symmetry asks for `appendMoved`. But that's just my 10 cents.

Reply:
how about removing it in this PR and so that we can move forward with this PR and adding it in another PR so it can be discussed separately?

Reply:
I'm not that emotionally attached to the idea, just wanted to give you a chance to reconsider :)

Reply:
I agree with salvipeter, theres no reason not to have `appendMoved` here.

Reply:
@timotheecour I'm readding it to get on with this and not subject salvipeter to this review ping pong :D
Removing `appendMoved` can also be done in a further PR (until the release); but as of now there is no argument to not add it.